src/capi-single-paidfor/web/index.js

```diff
@@ -59,25 +53,18 @@ function populateCard(host, responseJson) {
           ${icon}
           ${getValue('[%ArticleHeadline%]', responseJson.articleHeadline)}
         </h2>
-        <div class="advert__standfirst">
-            ${getValue('[%ArticleText%]', responseJson.articleText)}
-        </div>
       </div>
       <div class="advert__image-container">${generatePicture({
           url: imageUrl || responseJson.articleImage.backupSrc,
           classes: ['advert__image'],
           sources: responseJson.articleImage.sources
       })}</div>
     </a>
-    <a class="hide-until-mobile-landscape button button--large button--legacy-single" href="${clickMacro}${host}/[%SeriesUrl%]"  data-link-name="more" target="_top">
-      See more
-      ${arrowRight}
-    </a>
-    </div>
     <div class="badge js-badge">
       Paid for by
       <a class="badge__link" href="${clickMacro}${responseJson.branding.logo.link}" data-link-name="badge">
         <img class="badge__logo" src="${getValue('[%BrandLogo%]', responseJson.branding.logo.src)}" alt="">
       </a>
+    </div>
     </div>`;
```

Comment:
Minor if you want to - can we improve the HTML indentation for this block?

Reply:
nah

Reply:
Go on, then…